packages/admin/admin/src/dataGrid/CrudMoreActionsMenu.tsx

```diff
@@ -97,19 +91,13 @@ const MoreActionsButton = createComponentSlot(Button)<CrudMoreActionsMenuClassKe
 const MoreActionsMenuItem = createComponentSlot(MenuItem)<CrudMoreActionsMenuClassKey>({
     componentName: "CrudMoreActions",
     slotName: "menuItem",
-})(
-    css`
-        padding: 8px 15px 8px 30px !important;
-        column-gap: 10px;
-    `,
-);
+})();
 
 export function CrudMoreActionsMenu({ slotProps, overallActions, selectiveActions, selectionSize }: CrudMoreActionsMenuProps) {
     const {
         menu: menuProps,
         button: buttonProps,
         group: groupProps,
-        divider: dividerProps,
```

Comment:
The `dividerProps` should probably still be used and applied to both instances of `<CrudMoreActionsDivider />`, just like before this refactor.